lib/ovh.js

```diff
@@ -367,7 +403,19 @@ class Ovh {
       }
     }
 
-    if (path.indexOf('/auth') < 0) {
+    if (this.oauthConfig && (!this.accessToken || this.accessToken.expired())) {
```

Comment:
```suggestion
    if (this.oauthConfig && (!this.accessToken || this.accessToken.expired(10))) {
```

Since the lib allows it, adding a time buffer might prevents some race condition with expired tokens.
[go-ovh](https://github.com/ovh/go-ovh/blob/master/ovh/ovh_test.go#L613) has a 10 seconds one, suggesting the same.